libstdc++-v3/testsuite/18_support/contracts/handle_enforced_contract_violation.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2025 Free Software Foundation, Inc.
+//
+// This file is part of the GNU ISO C++ Library.  This library is free
+// software; you can redistribute it and/or modify it under the
+// terms of the GNU General Public License as published by the
+// Free Software Foundation; either version 3, or (at your option)
+// any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License along
+// with this library; see the file COPYING3.  If not see
+// <http://www.gnu.org/licenses/>.
+
+// { dg-options "-g0 -fcontracts -fcontracts-nonattr -fcontract-evaluation-semantic=observe" }
+// { dg-do run { target c++2a } }
+
+#include <exception>
+#include <cstdlib>
+#include <testsuite_hooks.h>
+#include <cstring>
+#include <iostream>
+
+#include "../../../include/std/contracts"
```

Comment:
I don't think you can do this - it will break testing of an installed compiler.
why not `#include <contracts>` ?